src/ott/neural/layers.py

```diff
@@ -22,118 +22,183 @@
 PRNGKey = jax.Array
 Shape = Tuple[int, ...]
 Dtype = Any
-Array = Any
+Array = jnp.ndarray
+
+# wrap to silence docs linter
+DEFAULT_KERNEL_INIT = lambda *a, **k: nn.initializers.lecun_normal()(*a, **k)
+DEFAULT_BIAS_INIT = nn.initializers.zeros
+DEFAULT_RECTIFIER = nn.activation.relu
 
 
 class PositiveDense(nn.Module):
-  """A linear transformation using a weight matrix with all entries positive.
+  """A linear transformation using a matrix with all entries non-negative.
 
   Args:
-    dim_hidden: the number of output dim_hidden.
-    rectifier_fn: choice of rectifier function (default: softplus function).
-    inv_rectifier_fn: choice of inverse rectifier function
-      (default: inverse softplus function).
-    dtype: the dtype of the computation (default: float32).
-    precision: numerical precision of computation see `jax.lax.Precision`
-      for details.
-    kernel_init: initializer function for the weight matrix.
-    bias_init: initializer function for the bias.
+    dim_hidden: Number of output dimensions.
+    rectifier_fn: Rectifier function. The default is
+      :func:`~flax.linen.activation.relu`.
+    use_bias: Whether to add bias to the output.
+    kernel_init: Initializer for the matrix. The default is
+      :func:`~flax.linen.initializers.lecun_normal`.
+    bias_init: Initializer for the bias. The default is
+      :func:`~flax.linen.initializers.zeros`.
+    precision: Numerical precision of the computation.
```

Comment:
is there a reason we still need this?

Reply:
The precision? Yes, for compatibility with `flax`, it also exposes it in `nn.Dense`.